### PR TITLE
MBST-13263: Added priority score reasons for the output json object.

### PR DESCRIPTION
**Pull request by MartinsIrbe**

### CODE REVIEW

File: atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java [hunk 1]
```diff
@@ -20,11 +20,7 @@
 import org.atlasapi.entity.ResourceRef;
 import org.atlasapi.output.FieldWriter;
 import org.atlasapi.output.OutputContext;
-import org.atlasapi.output.writers.CertificateWriter;
-import org.atlasapi.output.writers.LanguageWriter;
-import org.atlasapi.output.writers.ReleaseDateWriter;
-import org.atlasapi.output.writers.RestrictionWriter;
-import org.atlasapi.output.writers.SubtitleWriter;
+import org.atlasapi.output.writers.*;
```
Comment (Spartakovic) [inline atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java]: not good

File: atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java [hunk 2]
```diff
@@ -44,6 +41,7 @@ public ExtendedDescriptionAnnotation() {
         this.subtitleWriter = new SubtitleWriter(languageWriter);
         releaseDateWriter = new ReleaseDateWriter();
```
Comment (Spartakovic) [inline atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java]: why it not this.releaseDateWriter?

File: atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java (hunk 1 above)
Comment (MartinsIrbe) [inline atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java]: updated

File: atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java (hunk 2 above)
Comment (MartinsIrbe) [inline atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java]: Not sure as I haven't added this.

File: atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java (hunk 1 above)
Comment (timothy22000) [inline atlas-api/src/main/java/org/atlasapi/output/annotation/ExtendedDescriptionAnnotation.java]: Martins, check your IntelliJ settings to not let it import * when there is more than 5 imports from the same place. Ask me tomorrow if you haven't found out about it.

**Pull request merged**